api/build.gradle

```diff
@@ -53,7 +53,7 @@ dependencies {
     // https://app.snyk.io/vuln/SNYK-JAVA-COMGOOGLEGUAVA-32236
     implementation "com.google.guava:guava:27.0.1-jre"
     implementation "com.google.code.findbugs:annotations:3.0.1"
-    implementation "uk.gov.dhsc.htbhf:htbhf-request-context:0.0.2"
+    implementation "uk.gov.dhsc.htbhf:htbhf-common-rest:0.0.4"
```

Comment:
We can use this instead for internal dependencies:
`implementation "uk.gov.dhsc.htbhf:htbhf-common-rest:latest.release"`

Reply:
Updated to latest.release